arxiv_latex_cleaner/arxiv_latex_cleaner.py

```diff
@@ -151,11 +151,31 @@ def _remove_comments(content, parameters):
   content = _remove_environment(''.join(content), 'comment')
   for command in parameters['commands_to_delete']:
     content = _remove_command(content, command)
-  # If file ends with '\n' already, the split in last line would add an extra
-  # '\n', so we remove it.
-  return content.split('\n')
+  return content
```

Comment:
Why did you move this?

Reply:
I need a joined string in `_replace_tikzpictures` anyway so i performed the split after `_replace_tikzpictures`.
Otherwise I would need to add "\n" to all lines in content again, then joint them again in `_replace_tikzpictures`, and then split them again.
So I moved the content splitting part out of both functions, but yeah it's not that pretty that `_remove_comments` now consumes a content array of lines but returns a joined string.